ansible/roles/proteus-db/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Create database schema files
+  copy:
+    src: "{{ item }}"
+    dest: "{{ proteus_db_schema_path }}"
+    mode: 0644
+  with_items: "{{ proteus_db_files }}"
+
+- name: Run database schema commands
+  shell: "psql -U {{ proteus_db_user}} -h 127.0.0.1 -d {{ proteus_db_name }} -f {{ proteus_db_schema_path}}/{{ item }}"
+  environment:
+    PGPASSWORD: "{{ proteus_db_password }}"
+  with_items: "{{ proteus_db_files }}"
```

Comment:
IIRC, @bassosimone was struggling with sql updates embedded into the binary while launching concurrent processes as same DB is used by various services. So there was a suggestion to move SQL files away from the binary and apply them using some other tooling. Is it still the case?